FirebaseMessaging/Sources/Token/FIRMessagingCheckinStore.m

```diff
@@ -17,16 +17,18 @@
 #import "FirebaseMessaging/Sources/Token/FIRMessagingCheckinStore.h"
 
 #import "FirebaseMessaging/Sources/FIRMessagingCode.h"
+#import "FirebaseMessaging/Sources/FIRMessagingConstants.h"
 #import "FirebaseMessaging/Sources/FIRMessagingLogger.h"
 #import "FirebaseMessaging/Sources/FIRMessagingUtilities.h"
 #import "FirebaseMessaging/Sources/NSError+FIRMessaging.h"
-#import "FirebaseMessaging/Sources/Token/FIRMessagingAuthKeyChain.h"
+#import "FirebaseMessaging/Sources/Token/FIRMessagingAuthKeychain.h"
```

Comment:
I'm curious if using [GULKeychainUtils](https://github.com/google/GoogleUtilities/blob/main/GoogleUtilities/Environment/Public/GoogleUtilities/GULKeychainUtils.h) instead of `FIRMessagingAuthKeychain` may save some extra code and tests?

Reply:
Good call. I will take a look at.

Reply:
Let me know if you need any help with that.

Reply:
yeah maybe we should work on separate PR. Feel free to checkin in chen/fm-master while I keep adding more unit tests.

Reply:
sure